_source/lib/routes/server/api.coffee

```diff
@@ -113,6 +114,9 @@ createEndpoint = (collection, singular) ->
           return
 
         handler = Apollos[singular].update
+        if options.subDocName
```

Comment:
This section feels a little weird. It will create `Apollos[singular].update[child]` instead of `Apollos[singular][child].update` which is more convention to put the action last